docs/zh/deployment/distributed-deployment-guide.md

```diff
@@ -0,0 +1,1215 @@
+* [一、准备工作](#%E4%B8%80%E5%87%86%E5%A4%87%E5%B7%A5%E4%BD%9C)
```

Comment:
Now catalog can view by side bar.

Keep or delete those links?

Reply:
It looks like these links doesn't work when viewed in standalone website, so we may just delete them.